### PR TITLE
[AML] More log output to solve trac tickets

### PR DESCRIPTION
**Pull request by peak3d**
See title

## Types of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp [hunk 1]
```diff
@@ -1883,7 +1883,7 @@ int CAMLCodec::ReleaseFrame(const uint32_t index, bool drop)
   if (drop)
     vbuf.flags |= V4L2_BUF_FLAG_DONE;
 
-  CLog::Log(LOGDEBUG, LOGVIDEO, "CAMLCodec::ReleaseFrame idx:%u", index);
+  CLog::Log(LOGDEBUG, LOGVIDEO, "CAMLCodec::ReleaseFrame idx:%u, drop:%d", index, (int)drop);
```
Comment (Rechi) [inline xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp]: C++ style cast

File: xbmc/cores/VideoPlayer/DVDCodecs/Video/AddonVideoCodec.cpp [hunk 2]
```diff
@@ -241,6 +241,7 @@ CDVDVideoCodec::VCReturn CAddonVideoCodec::GetPicture(VideoPicture* pVideoPictur
 
     return CDVDVideoCodec::VC_PICTURE;
   case VIDEOCODEC_RETVAL::VC_EOF:
+    CLog::Log(LOGINFO, "CAddonVideoCodec: GetPicture: EOF"
```
Comment (Rechi) [inline xbmc/cores/VideoPlayer/DVDCodecs/Video/AddonVideoCodec.cpp]: closing bracket is missing

**Comment by peak3d** [comment]
Huh, this was a really lame PR, will fix :-)

**Comment by peak3d** [comment]
@rechi ok with it?

**Comment by Rechi** [comment]
Yes, looks good now

**Pull request merged**